keiko-redis-spring/keiko-redis-spring.gradle

```diff
@@ -9,12 +9,13 @@ dependencies {
   testImplementation "org.springframework:spring-test"
   testImplementation "org.springframework.boot:spring-boot-test"
   testImplementation "org.assertj:assertj-core"
+  testImplementation("io.spinnaker.kork:kork-jedis-test")
 
   testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
 }
 
 test {
   useJUnitPlatform {
-    includeEngines "jupiter"
+    includeEngines "junit-jupiter"
```

Comment:
Same here.

Reply:
Verified by removing the `test { }`. Test executed and passed.